pkg/config/config.go

```diff
@@ -194,6 +197,58 @@ func (c Config) String() string {
 	return string(b)
 }
 
+// GetJiraFieldKey returns the jira key associated to a field.
+func (c *Config) GetJiraFieldKey() error {
```

Comment:
Do you mind moving this function out of the config.go? It would be epic to separate JIRA API integrations vs config logic. Also let's avoid extra config state that does not belong to user config directly - it's ok to store this fieldkey as variable somewhere.

Also probably you mean `ValidateJIRAField` or something - it's ok to have `validateXYZ` function in cmd/ somewhere 🤔